DevEdu.API/DevEdu.DAL/Repositories/NotificationRepository.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using Dapper;
+using DevEdu.DAL.Models;
+namespace DevEdu.DAL.Repositories
+{
+    public class NotificationRepository : BaseRepository, INotificationRepository
+    {
+        public int AddNotification(NotificationDto notificationDto)
+        {
+            return _connection.QuerySingle<int>(
+                "dbo.Notification_Insert",
```

Comment:
please move all the stored procedures' names to the top of this file (like as we did it in the lesson)